app.py

```diff
@@ -1,641 +1,44 @@
-                    flavorIds.append(ObjectId(j["_id"]["$oid"]))
-        else:
-            flavorIds.append(
-                ObjectId(add_flavor_return_id(i))
-            )
-
-    ingredientIds = []
-    for i in dataDict["ingredients"]:
-        if any(j["name"] == i["name"] for j in ingredients):
-            for j in ingredients:
-                if j["name"] == i["name"]:
-                    ingredients_id = j["_id"]["$oid"]
-        else:
-                ingredients_id = add_ingredient_return_id(i["name"], i["type"])
-
-        ingredientIds.append(
-            {
-                "ingredient": ObjectId(ingredients_id),
-                "quantity": i['quantity'],
-                "units": i['units'],
-                "type": i['type']
-            })
-
-    connection = mongo_connect(mongo_uri)
-
-    connection["cocktails"].update_one(
-        {"_id": ObjectId(dataDict["id"])},
-        {"$set": 
-            {"name": dataDict["name"],
-            "description": dataDict["description"],
-            "flavor_tags": flavorIds,
-            "ingredients": ingredientIds,
-            "method": dataDict["instructions"],
-            "glass": dataDict["glass"],
-            "equipment": dataDict["equipment"],
-            "creator": ObjectId(session['_id']),
-            "updated_at": str(datetime.now()),
-            "preferred_spirits": [],
-            "image_url": dataDict["image_url"]}
-        }
-    )
-    return "success"
+app = Flask(__name__)
 
-#delete routes
-@app.route('/d/delete_cocktail', methods=["POST"])
-def delete_cocktail():
-    """
-    Method to delete cocktail from the database
-    """
-    data = request.data
-    cocktail = json.loads(data)
-    print(cocktail)
-    connection = mongo_connect(mongo_uri)
-    connection["cocktails"].delete_one({
-        "_id":ObjectId(cocktail["object_id"])
-    })
-    return "done!"
+# os.environ['MONGO_URI'] = 
+# mongo_uri = os.environ.get('MONGO_URI')
+app.secret_key = "asdfsdf"
+# app.secret_key = os.environ.get('SECRET_KEY')
 
-#search routes
+app.register_blueprint(home)
+app.register_blueprint(login_logout)
+app.register_blueprint(api)
+app.register_blueprint(filters)
+app.register_blueprint(profiles)
+app.register_blueprint(add_drink)
+app.register_blueprint(comment)
+app.register_blueprint(update)
+app.register_blueprint(delete)
+app.register_blueprint(searchs)
 
-@app.route('/s/<type>/<terms>', methods=["GET"])
-def search(type, terms):
-    fieldName = ""
-    connection = mongo_connect(mongo_uri)
-    if type == "users":
-        fieldName += "user"
-    fieldName += "name"
-    results = connection[type].find(
-        {fieldName: {'$regex': terms, '$options': 'i'}}
-        )
-    return dumps(results)
 
 if __name__ == '__main__':
-    port = int(os.environ.get("PORT", 33507))
-    app.run(host='0.0.0.0', port=port)  
-    #app.run(debug="true")
+    # port = int(os.environ.get("PORT", 33507))
+    # app.run(host='0.0.0.0', port=port)
+    app.run(debug="true")
```

Comment:
DEBUG should never be true for production as it poses security risks so you should have some logic to make it true in development and false in production